scipy/stats/_bws_test.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+from scipy import stats
+from functools import partial
+from collections import namedtuple
+from ._axis_nan_policy import _axis_nan_policy_factory
+
+
+def bws_statistic(x, y, alternative, method):
```

Comment:
It's ok that you removed `axis` for now. Much slower, but let's keep this as simple as possible.